src/repositories/RemittanceRepository.js

```diff
@@ -153,5 +153,8 @@ export const RemittanceSettingsRepository = {
     get: service + 'getCGP',
     set: service + 'setCGP',
     del: service + 'delCGP'
+  },
+  ExtraIncome: {
```

Comment:
merge from master and check if this endpoint already present.
if it's please remove yours

Reply:
not alrady present